docker/Dockerfile

```diff
@@ -20,14 +20,16 @@ LABEL description="Precompiled Docker image for Composable" \
       image.documentation="https://github.com/ComposableFi/composable#readme"
 
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
-RUN curl "https://storage.googleapis.com/composable-binaries/community-releases/${VERSION}/composable-${VERSION}.tar.gz" | tar -xz \
-    chmod +x target/release/composable && mv target/release/composable /usr/local/bin \
-    useradd -m -u 1001 -U -s /bin/sh -d /composable composable && \
-    mkdir -p /composable/.local/share/local_testnet && \
-    mkdir /data && \
-    chown -R composable:composable /data && \
-    chown -R composable:composable /composable/.local/share/local_testnet && \
-    ln -s /data /composable/.local/share/local_testnet 
+
+RUN curl "https://storage.googleapis.com/composable-binaries/community-releases/${VERSION}/composable-${VERSION}.tar.gz" | tar -xz 
+RUN ls
+RUN chmod +x target/release/composable && mv target/release/composable /usr/local/bin
+RUN useradd -m -u 1001 -U -s /bin/sh -d /composable composable
+RUN mkdir -p /composable/.local/share/local_testnet
+RUN mkdir /data
+RUN chown -R composable:composable /data
+RUN chown -R composable:composable /composable/.local/share/local_testnet
+RUN ln -s /data /composable/.local/share/local_testnet 
```

Comment:
Also, let's compbine all the commands here to a single line

Reply:
The single line was what was making it fail in the first place, but I will try it again and see